smoke-tests/src/test/groovy/io/opentelemetry/smoketest/LibertySmokeTest.groovy

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.smoketest
+
+import io.opentelemetry.proto.trace.v1.Span
+import okhttp3.Request
+import spock.lang.Shared
+import spock.lang.Unroll
+
+import java.util.jar.Attributes
+import java.util.jar.JarFile
+
+class LibertySmokeTest extends AppServerTest {
```

Comment:
Should some (most?) of these tests live in `AppServerTest`? And tested for all app servers?

Reply:
Yes they should. I plan to do it a bit later along with changing the tests for all the servers so that each configuration starts server once and runs all the tests.
Though for this to happen it would be really helpful if all servers behaved the same way or at least there was a clear understanding what span names they should use.